sidebars.js

```diff
@@ -414,6 +414,10 @@ module.exports = {
                   "Variable labeling",
                   "components/userguide/additional-features/variable-labeling/"
                 ),
+                optimizeLink(
+                  "Data source variations",
```

Comment:
```suggestion
                  "Data source variants",
```

I think this is a typo 😄